Assets/Scripts/GameController.cs

```diff
@@ -330,7 +371,7 @@ public class GameController : MonoBehaviour {
 
 	// Runs every frame
 	void Update() {
-		// Testing move/zoom
+		// Computer move/zoom
```

Comment:
it won't :(
Mouse events will work on phone but not vice verca, and also each is optimized for its respective platform so I'd like to keep it separate